src/elements/file-selector/file-selector-dropzone/file-selector-dropzone.ts

```diff
@@ -0,0 +1,132 @@
+import { type CSSResultGroup, LitElement, type TemplateResult } from 'lit';
+import { property, customElement } from 'lit/decorators.js';
+import { ref } from 'lit/directives/ref.js';
+import { html } from 'lit/static-html.js';
+
+import type { SbbSecondaryButtonStaticElement } from '../../button.js';
+import { forceType, slotState } from '../../core/decorators.js';
+import { i18nFileSelectorButtonLabel, i18nFileSelectorSubtitleLabel } from '../../core/i18n.js';
+import { fileSelectorCommonStyle, SbbFileSelectorCommonElementMixin } from '../common.js';
+
+import '../../button/secondary-button.js';
+import '../../button/secondary-button-static.js';
+import '../../icon.js';
+
+import style from './file-selector-dropzone.scss?lit&inline';
+
+/**
+ * It allows to select one or more file from storage devices via button click or drag and drop, and display them.
+ *
+ * @slot error - Use this to provide a `sbb-form-error` to show an error message.
+ * @event {CustomEvent<Readonly<File>[]>} fileChanged - An event which is emitted each time the file list changes.
+ * @event change - An event which is emitted each time the user modifies the value. Unlike the input event, the change event is not necessarily fired for each alteration to an element's value
+ * @event input - An event which is emitted each time the value changes as a direct result of a user action.
+ */
+export
+@customElement('sbb-file-selector-dropzone')
+@slotState()
+class SbbFileSelectorDropzoneElement extends SbbFileSelectorCommonElementMixin(LitElement) {
+  public static override styles: CSSResultGroup = [fileSelectorCommonStyle, style];
+  public static readonly events = {
+    fileChangedEvent: 'fileChanged',
+  } as const;
+
```

Comment:
The stories of the two components need the event in the action.handles prop.
Since the event is defined in the inner class of the mixin, I can't reach it in any other way than redefining the events.

Reply:
should it maybe be in the mixintype? so it's better linked?

Reply:
i have done some test, apparently the static properties are somehow not considered: even if i add the prop in the mixin, it doesn't want the override keyword in the extended class :|

Reply:
ok. Then I have to live with :)